### ISSUE TITLE
2 loot windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#234]
```
On occasion I am seeing multiple loot windows


What steps will reproduce the problem?
1. Defeat monster.
2. 2 loot windows appear.
3. Click "pickup" in both windows and they both close.  User is not credited 
with twice the loot which is correct.

What is the expected output? What do you see instead?
See only one loot window.

What version of the product are you using? On what device?
0.6.9
DROIDX 2.3

Please provide any additional information below.


```

Original issue reported on code.google.com by `jgemmajr@gmail.com` on 10 Aug 2011 at 2:18

**Comment by GoogleCodeExporter** [comment]
```
I find this happens when I accidently click very quickly. I defeat the 
montster, and then move onto the lootbag *before* the autopickup triggers; the 
autopickup then triggers, leaving me with two loot windows.
```

Original comment by `casey...@gmail.com` on 22 Oct 2012 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.6.11
```

Original comment by `sdeva...@gmail.com` on 9 Feb 2013 at 5:34
- Changed state: **Fixed**
